### PR TITLE
remove legacy recovery tests

### PR DESCRIPTION
**Pull request by tjungblu**
cleaning up the old tests and the bastion ssh code

### CODE REVIEW

File: test/extended/util/annotate/generated/zz_generated.annotations.go [hunk 1]
```diff
@@ -671,12 +671,6 @@ var Annotations = map[string]string{
 
 	"[sig-arch][Early] Managed cluster should [apigroup:config.openshift.io] start all core operators": " [Skipped:Disconnected] [Suite:openshift/conformance/parallel]",
 
-	"[sig-arch][Feature:ClusterUpgrade] Cluster should be upgradeable after finishing upgrade [Late][Suite:upgrade]": "",
-
-	"[sig-arch][Feature:ClusterUpgrade] Cluster should be upgradeable before beginning upgrade [Early][Suite:upgrade]": "",
```
Comment (hasbro17) [inline test/extended/util/annotate/generated/zz_generated.annotations.go:676]: This is not related to our cleanup right? And the test still seems to be there:
https://github.com/openshift/origin/blob/2d41e5256e5642b858a2defa3894a9922f839e87/test/e2e/upgrade/upgrade.go#L142-L146

We might want to double check if we aren't disabling this unintentionally.

Comment (tjungblu) [reply test/extended/util/annotate/generated/zz_generated.annotations.go:676]: good point, no idea how that got into it...

Comment (tjungblu) [reply test/extended/util/annotate/generated/zz_generated.annotations.go:676]: okay that's a really weird issue, it's a completely different folder. On the master branch I don't have that change generated.

Comment (tjungblu) [reply test/extended/util/annotate/generated/zz_generated.annotations.go:676]: okay that's a really weird issue, it's a completely different folder. On the master branch I don't have that change generated.

**Comment by hasbro17** [comment]
/approve
Needs a go mod vendor update I think.
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/27917/pull-ci-openshift-origin-master-verify-deps/1658045980387315712

**Comment by tjungblu** [comment]
@bparees could you please take a look when you have some time?
cc @soltysh because I also see those touching some vendor k8s tests

We have successfully replaced the existing tests with two new presubmits:
https://sippy.dptools.openshift.org/sippy-ng/tests/Presubmits/analysis?test=%5Bsig-etcd%5D%5BFeature%3ADisasterRecovery%5D%5BSuite%3Aopenshift%2Fetcd%2Frecovery%5D%5BTimeout%3A2h%5D%20%5BFeature%3AEtcdRecovery%5D%5BDisruptive%5D%20Recover%20with%20snapshot%20with%20two%20unhealthy%20nodes%20and%20lost%20quorum%20%5BSerial%5D

https://sippy.dptools.openshift.org/sippy-ng/tests/Presubmits/analysis?test=%5Bsig-etcd%5D%5BFeature%3ADisasterRecovery%5D%5BSuite%3Aopenshift%2Fetcd%2Frecovery%5D%5BTimeout%3A30m%5D%20%5BFeature%3AEtcdRecovery%5D%5BDisruptive%5D%20Restore%20snapshot%20from%20node%20on%20another%20single%20unhealthy%20node%20%5BSerial%5D

which are running now, which means we can clean this old bastion-ssh based tests up :)

**Comment by soltysh** [comment]
/hold 
to merge https://github.com/openshift/origin/pull/27894 first

**Comment by soltysh** [comment]
/hold cancel
the other merged
@tjungblu you'll need to rebase, sorry about it :sweat:
